src/schema/rules/datatypes/meg.yaml

```
@@ -60,8 +66,10 @@
 - suffixes:
   - headshape
   extensions:
-  - .*
+  - "*"
```

I think this was a merge conflict resolution? Seems out of scope.

```suggestion
  - .*
```

---

Ah, yes. I think I just merged it badly.